docs/HACKING.md

```diff
@@ -1,158 +1,160 @@
-       When building the operator locally, the WMCO binary is created as part of the operator image build process and
-       can be found at `build/_output/bin/windows-machine-config-operator`, this is the default value of this option.
-- `-t` specify the test to run. All tests are run if the option is not used. The allowed options are:
-  - `all` all the tests are run
-  - `basic` creation, network and deletion tests are run
-  - `upgrade` creation, upgrade, reconfiguration and deletion tests are run
+The operator can now be deployed
+[Manually](#deploying-the-operator-manually),
+[Using the OLM](#running-the-olm-hack-script),
+[Using the e2e tests](#running-the-e2e-tests),
+[Using the e2e tests on platform agnostic infrastructure](#running-e2e-tests-on-platform-agnostic-infrastructure), or
+[Using the official deployment method](#deploying-Using-the-official-deployment-method).
+
+### Deploying the operator manually
 
-Example command to run the full test suite with 2 instances configured by the Windows Machine controller, and 1
-configured by the ConfigMap controller, skipping the deletion of all the nodes.
 ```shell script
-hack/run-ci-e2e-test.sh -s -m 2 -c 1
+make deploy IMG=$OPERATOR_IMAGE
 ```
-Please note that you do not need to run `hack/olm.sh run` before `hack/run-ci-e2e-test.sh`.
 
-#### Running e2e tests on platform-agnostic infrastructure
+The above command will directly deploy all the resources required for the operator to run, as well as the WMCO pod.
 
-To run the WMCO e2e tests on a bare metal or other platform-agnostic infrastructure (platform=none), where there
-is no cloud provider specification, the desired number of Windows instances must be provisioned beforehand, and the
-instance(s) information must be provided to the e2e test suite through the `WINDOWS_INSTANCES_DATA` environment
-variable.
+#### Cleaning up a manual deployment  
 
-To deploy machines using the [infrastructure providers supported by WMCO](wmco-prerequisites.md#supported-cloud-providers-based-on-okdocp-version-and-wmco-version),
-refer to the specific provider documentation. See Windows instance [pre-requisites](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/byoh-instance-pre-requisites.md).
+To remove the installed resources:
 
-The information you need to collect from each Windows instance is:
-- the internal IP address
-- the Windows administrator username
+```shell script
+make undeploy
+```
 
-Export `WINDOWS_INSTANCES_DATA` as an environment variable with the corresponding [windows-instances ConfigMap](https://github.com/openshift/windows-machine-config-operator#adding-instances)
-data section, and a new `windows-instances` ConfigMap will be created during the execution of the e2e test suite.
+### Deploying using the OLM 
 
-For example:
-```shell
-export WINDOWS_INSTANCES_DATA='
-data:
-  10.1.42.1: |-
-    username=Administrator
-'
+Deploy the operator lifecycle manager onto your cluster with the hack script.
```

Comment:
You are not deploying OLM. OLM is part of the payload and runs on all clusters by default.